tests/composite_pk/models/tenant.py

```diff
@@ -0,0 +1,50 @@
+from django.db import models
+
+
+class Tenant(models.Model):
+    name = models.CharField(max_length=10, default="", blank=True)
+
+
+class Token(models.Model):
+    pk = models.CompositePrimaryKey("id")
```

Comment:
Do we want to allow singleton composite primary keys?

Reply:
Yes they are allowed.
I don't really have an opinion on whether we should allow them or not. I don't have any arguments for or against it.
If you think we shouldn't, we can disable them.